fe/fe-core/src/main/java/com/starrocks/storagevolume/storageparams/HDFSStorageParams.java

```diff
@@ -0,0 +1,70 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.starrocks.storagevolume.storageparams;
+
+import com.starrocks.storagevolume.StorageVolume;
+import com.starrocks.storagevolume.credential.hdfs.HDFSCredential;
+import com.starrocks.storagevolume.credential.hdfs.HDFSKerberosCredential;
+import com.starrocks.storagevolume.credential.hdfs.HDFSSimpleCredential;
+
+import java.util.Map;
+
+public class HDFSStorageParams implements StorageParams {
+    public static final String HDFS_AUTHENTICATION = "hadoop.security.authentication";
+    public static final String HDFS_USER_NAME = "username";
```

Comment:
need to consider the unified params name convention. e.g. all hdfs params starts with `hdfs.`, similar to what aws params starts with `aws.`

Reply:
> need to consider the unified params name convention. e.g. all hdfs params starts with `hdfs.`, similar to what aws params starts with `aws.`

This refers to https://docs.starrocks.io/zh-cn/latest/sql-reference/sql-statements/data-manipulation/BROKER%20LOAD#hdfs

Reply:
ok, prompt this to PM to make the final decision, whether to unify the parameters name convention. or keep as it.

Reply:
> ok, prompt this to PM to make the final decision, whether to unify the parameters name convention. or keep as it.

We will keep as it temporarily.